subprojects/docs/docs.gradle

```diff
@@ -415,7 +415,7 @@ tasks.addRule("view«Doc Task Name» - Opens entry point") { String taskName ->
             tasks.create(taskName) {
                 dependsOn task
                 doLast {
-                    Class.forName("java.awt.Desktop").newInstance().browse(file(task.entryPoint).toURI())
+                    Class.forName("java.awt.Desktop").getConstructor().newInstance.browse(file(task.entryPoint).toURI())
```

Comment:
We use now `ClassUtils` and `getConstructor().newInstance()` for the same class in the same file. I think we should decide on something. If `getConstructor().newInstance()` works, I would go for that.

Reply:
Yes, for the classes we can control, I use `getConstructor().newInstance()` as much as possible.